extide/gradle/src/org/netbeans/modules/gradle/customizer/Bundle.properties

```diff
@@ -54,3 +54,27 @@ BuildActionsCustomizer.btnDisableAction.text=&Disable
 BuildActionsCustomizer.jLabel2.text=The action is disabled in this configuration.
 BuildActionsCustomizer.btnRestore.text=&Restore
 BuildActionsCustomizer.btnRemove2.text=Re&set
+GradleExecutionPanel.cbAugmentedBuild.text=&Augmented Build Execution
+GradleExecutionPanel.lbIncludeOpenProjects.text=<html>Create a composite project from this project and the other open Gradle projects by generating '--include-build' parameters.
+GradleExecutionPanel.cbIncludeOpenProjects.text=Include Open Projects
+GradleExecutionPanel.lbTrustTerms.text=<html><p>Executing Gradle can be potentially un-safe as it allows arbitrary code execution.</p>\n<p/>\n<p>By trusting this project, and with that all its subprojects, you entitle NetBeans to invoke Gradle to load project details without further confirmation.</p><p></p><p>Invoking any build related actions, would mark this project automatically trusted.</p>
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
```

Comment:
Something weird going on with license here?!

Reply:
It seems the form designer took the license header as a comment belonging to the first property item, when I did copy+paste between the panels. Fixed now.